src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/CreateArticleModalComponentObject.java

```diff
@@ -96,4 +96,7 @@ public boolean isCreateNewArticleModalVisible() {
     }
   }
 
+  public void close() {
```

Comment:
what if element was already closed?
might be confusing when we'll be expanding these tests

Reply:
@hakubo if it is closed, then test will fail because close button will not be found